beluga_ros/test/test_occupancy_grid.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <beluga_ros/occupancy_grid.hpp>
+
+#if BELUGA_ROS_VERSION == 1
+#include <boost/smart_ptr.hpp>
+#endif
+
+namespace {
+
+TEST(TestOccupancyGrid, Instantiation) {
+  constexpr std::size_t kWidth = 100;
+  constexpr std::size_t kHeight = 200;
+
+#if BELUGA_ROS_VERSION == 2
+  auto message = std::make_shared<beluga_ros::msg::OccupancyGrid>();
+#elif BELUGA_ROS_VERSION == 1
+  auto message = boost::make_shared<beluga_ros::msg::OccupancyGrid>();
+#else
+#error BELUGA_ROS_VERSION is not defined or invalid
+#endif
```

Comment:
@nahueespinosa meta: should we have a `beluga_ros::utility` namespace to hide this?

Reply:
For now, this is the only place I've had to do this, just for a test, so I'd rather wait and see if we need to implement our own `utility` namespace yet.

Reply:
Sounds reasonable to me.